cmd/server/flags.go

```diff
@@ -473,3 +473,14 @@ var flags = append([]cli.Flag{
 		Usage:   "Flag to decrypt all encrypted data and disable encryption on server",
 	},
 }, logger.GlobalLoggerFlags...)
+
+// If woodpecker is running inside a container the default value for
+// the datasource is different from running outside a container.
+func datasourceDefaultValue() string {
+	_, found := os.LookupEnv("WOODPECKER_IN_CONTAINER")
+	if found {
+		return "/var/lib/woodpecker/woodpecker.sqlite"
+	} else {
+		return "woodpecker.sqlite"
```

Comment:
On the other hand, containers doesn't follow standard sometimes and place data in `/data`, `/bitnami/mongodb`, etc. directories.

Reply:
The current "default" when running inside the container is like it is and I do not want to change that.

Reply:
I think this is totally fine as `./woodpecker.sqlite` is mainly used for development and similar. Most users are using the container images or some dpkg, nixpack, ... defining other default paths anyways.